RELEASE_NOTES.md

```diff
@@ -1,5 +1,113 @@
 # Release Notes
 
+# Staging 2019/06/05
+## Ingest
+
+## Upload
+
+## Data Store
+Version: 74b216e20ba34156dcd2d1c17469296200dadb6e
+* [easy] Use the same regex for all uuids. (#2185)
+* Retry V2 notifications every 6 hours for 3 days (#2169)
+* Miscellaneous fixes (#2188)
+* Revert "Begin DSS builder (#1149)" (#2187)
+* Add operation to trigger sync (#2180)
+* Use sqs batch send from dcplib (#2179)
+* bump requirements (#2186)
+* Remove notifier-v2 retries for non-existent objects (#2183)
+* Remove indexer retries for non-existent objects (#2182)
+* Add operation to verify entity replication (#2178)
+* Lambda Layers (#2168)
+* Lambda Tagging (#2150)
+
+
+## Secondary Analysis
+### Pipeline-tools v0.52.0
```

Comment:
We should update the version in the components.yml file as well right? https://github.com/HumanCellAtlas/dcp/blob/staging/components.yml#L18

Reply:
good catch. Updated